lib/web_ui/test/golden_tests/engine/screenshot.dart

```diff
@@ -0,0 +1,43 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
We already have `test/golden_tests/engine/scuba.dart` and `EngineScubaTester` that contains all this stuff. Is there a reason for the duplication?

Reply:
Its a start to dedup a lot of test code. There are only 4 tests using scuba and DomCanvas soon going away. BitmapCanvas will write out DOM nodes for rects etc soon as well and desire to run canvaskit+html tests in a uniform way. Plan to move scuba bits into screenshot and clean things up across all tests.